accounts/admin.py

```diff
@@ -1,3 +1,35 @@
 from django.contrib import admin  # noqa
 
 # Register your models here.
+from accounts.models import User
+
+
+class UserAdmin(admin.ModelAdmin):
```

Comment:
You can use a decorator. It's more clear.

```
@admin.register(User)
```

Reply:
I'm agree. I'll change it on next commit.